app/controllers/application_controller.rb

```diff
@@ -1,7 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :cart, :current_user
+  helper_method :cart
+  helper_method :current_user
```

Comment:
Can we also concatenate these methods into a single line, like
`helper_method :cart, :current_user, ...` ?